server/api/controllers/image-controller.ts

```diff
@@ -44,42 +43,56 @@ export class ImageController {
   /*
   Handles the /api/image/upload endpoint 
   - handles all possible errors with the file upload (no file, no fileype, not recognized filetype)
-  - if validation passes, adds the request to the ImageRequestQueue
+  - if validation passes, adds the request with all uploaded files to the ImageRequestQueue
   */
-  static uploadImage(req: Request, res: Response) {
-    try {
-      const image = req.files?.image as UploadedFile;
 
-      if (!image) {
-        throw new Error("No image was uploaded");
+  static uploadImages(req: Request, res: Response) {
+    try {
+      // this is done because express-upload has no way to tell you whether one file or multiple files were uploaded
+      // casting singular file to an array, or keeping multiple files as an existing array
+      const images = (req.files?.images as UploadedFile[]).length
```

Comment:
nit: we're not using `Array.isArray`?

Reply:
thats the function i was lookin for, its been changed